new/LIT7247QeneSemaetena.xml

```diff
@@ -56,12 +56,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
     <text>
         <body>
             <div type="edition" xml:lang="gez">
-                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/>, fols 56rb. Division into verses is indicated with ። . 
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/>, fol. 56rb. Division into verses is indicated with ። . 
                     The end of the poem is indicated with ። ። .</note>
                 <ab>
                     <l n="1">ሰማዕትነ፡ ገላውዴዎስ፡ ነዳየ፡ ዘመን፡ ወመዋዕል፡ እንዘ፡ ይረውፅ፡ አልፀቀ፡ እስከ፡ አንቀጸ፡ ቄሳር፡ ገድል።</l> 
                     <l n="2">ወተርኅወ፡ ሎቱ፡ ሙባኡ፡ መስቀል።</l>
-                    <l n="3">ኀበ፡ ፍርፋረ፡ ሞት፡ ተርኅወ፡ ወሰቲ፡ መከራ፡ ጥሉል።</l>
+                    <l n="3">ኀበ፡ ፍርፋረ፡ ሞት፡ ተርኅወ፡ ወሰቴ፡ መከራ፡ ጥሉል።</l>
```

Comment:
I think it must be a a sixth-order ስ, but if you think otherwise, it's fine as it is :)